src/e2e-test/java/io/cdap/plugin/spanner/stepsdesign/SpannerSource.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.spanner.stepsdesign;
+
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.common.stepsdesign.TestSetupHooks;
+import io.cdap.plugin.spanner.actions.SpannerActions;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+/**
+ * Spanner source plugin related test step definitions.
+ */
+public class SpannerSource implements CdfHelper {
+
```

Comment:
Indent looks incorrect. You can use [this](https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard) Intellij settings file. Please reformat all files in PR.